Json.More/JsonElementExtensions.cs

```diff
@@ -181,8 +182,8 @@ public static JsonElement AsJsonElement(this short value)
 		/// <remarks>This is a workaround for lack of native support in the System.Text.Json namespace.</remarks>
 		public static JsonElement AsJsonElement(this bool value)
 		{
-			var doc = JsonDocument.Parse($"{value.ToString().ToLower()}");
-			return doc.RootElement;
+			using var doc = JsonDocument.Parse(value ? "true" : "false");
```

Comment:
This could further be optimized by returning the same static true or false instances of `JsonElement`.